.github/workflows/go-check.yaml

```diff
@@ -0,0 +1,47 @@
+name: Go related checks
+
+on:
+  pull_request: {}
+  push:
+    branches:
+      - master
+
+jobs:
+  go-mod:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14.2
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Check module vendoring
+        run: |
+          go mod tidy
+          go mod vendor
+          git diff --exit-code
+
+  generate-k8s-api:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14.2
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          path: src/github.com/${{ github.repository }}
+      - name: Install protobuf dependencies
+        run: |
+          sudo apt install -y protobuf-compiler libprotobuf-dev
```

Comment:
Might be worth set a specific protobuf version.

Reply:
Sure, I updated with the same version in vagrant box, hope it is fine :)

Reply:
@aanm as discussed, I have updated protobuf version to 3.11.4. Please let me know if you have other comment.